config/kafkaConfig.js

```diff
@@ -1,63 +1,81 @@
 //dependencies
-const kafka = require('kafka-node')
-
-var connect = function() {
-
-    Producer = kafka.Producer
-    KeyedMessage = kafka.KeyedMessage
-    client = new kafka.KafkaClient({
-      kafkaHost : process.env.KAFKA_URL
-    })
-
-    client.on('error', function(error) {
-      console.error.bind(console, "kafka connection error!")
+const kafka = require("kafka-node");
+
+var connect = function () {
```

Comment:
@ankitpws Simplify this and optimize this - an array of topics to listen to and the corresponding consumer file to pass the message to, it should be the same across all services also.